### PR TITLE
Update readme to reflect new/live version of server

### PR DESCRIPTION
**Pull request by GreysonElkins**
#### What's this PR do?
* updates the readme to reflect the live version of the server
#### Any background context you want to provide?
The new BE is a copy & paste job of V1 with adjustments made to migrations primarily, but also dependencies and environmental variables. This may not reflect who actually wrote which lines of code any longer. I've only linked the new copy here to keep things simple, but if y'all would like to adjust this, by all means, let's! 
The new BE repo's readme does link to V1 and credit you all as contributors! We can adjust that readme as well of course!

### CODE REVIEW

File: src/ApiHelper/ApiHelper.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const apiHead = 'http://localhost:3005/api/v1'
+const apiHead = 'http://localhost:3005/api/v2'
```
Comment (Abdeboskey) [inline src/ApiHelper/ApiHelper.js:1]: V2 alright!

**Review by Abdeboskey — APPROVED** [review]
Looks good, thank you for working on this!

File: src/ApiHelper/ApiHelper.js (hunk 1 above)
Comment (carlymclift) [reply src/ApiHelper/ApiHelper.js:1]: 👍🏼

**Review by carlymclift — APPROVED** [review]
Approved!

**Comment by nickhartdev** [comment]
Looks great! Thanks for doing this!

**Pull request merged**